src/api/providers/openai.ts

```diff
@@ -14,7 +14,12 @@ export class OpenAiHandler implements ApiHandler {
 	constructor(options: ApiHandlerOptions) {
 		this.options = options
 		// Azure API shape slightly differs from the core API shape: https://github.com/openai/openai-node?tab=readme-ov-file#microsoft-azure-openai
-		if (this.options.openAiBaseUrl?.toLowerCase().includes("azure.com")) {
+		if (this.options.openAiBaseUrl?.toLowerCase().includes("deepseek")) {
```

Comment:
Let's use this instead.

```suggestion
		if (this.options.openAiBaseUrl?.toLowerCase().includes("azure.com") && !this.options.openAiModelId?.toLowerCase().includes("deepseek")) {
```

We shouldn't need to duplicate the logic with this change.

Reply:
@dcbartlett 
Thank you for pointing that out. Fixed.